src/app/src/components/Navbar/AuthMenu.jsx

```diff
@@ -24,13 +24,13 @@ import { convertFeatureFlagsObjectToListOfActiveFlags } from '../../util/util';
 
 const styles = {
     profileButton: {
-        color: 'white',
```

Comment:
We have an existing `COLORS.js`  so where it makes sense we could introduce a constant and make future tweaking easier.

Reply:
I think I actually looked for `COLORS.js` on my own once and missed it, since it's currently `COLOURS.js` 🇬🇧